packages/react-scripts/scripts/start.js

```diff
@@ -12,6 +12,9 @@
 process.env.BABEL_ENV = 'development';
 process.env.NODE_ENV = 'development';
 
+// We want to run this script on localhost with https on by default
+process.env.HTTPS = process.env.HTTPS || 'true';
```

Comment:
Check `=== undefined` or else we'll never be able to turn this off, right?

Reply:
Good catch